strawberry/contrib/django/type.py

```diff
@@ -0,0 +1,74 @@
+import strawberry
+from strawberry.contrib.django.converter import convert_django_field
+from strawberry.type import type as strawberry_type
+
+
+def model_type(
+    cls=None,
+    *,
+    model=None,
+    exclude_fields=[],
+    only_fields=[],
```

Comment:
I would say for security's sake: please do not add an `exclude_fields` option and support only `fields`, the list of included fields. So often internal fields are added to models and get accidentally shown on a Django ModelForm  or a DRF API endpoint, because they used their "all fields except these" mode. Please only support a list of fields, and if users want to show them on the API make them go the small extra step of listing them explicitly.

Reply:
Good call. I've modified the `model_type` decorator to match the way that Django handles form fields ([docs](https://docs.djangoproject.com/en/2.2/topics/forms/modelforms/#selecting-the-fields-to-use)). By adding a `field` parameter which specifies a list of fields to use, along with a special value of `'__all__'` that will expose all fields, we mitigate the security risk of accidentally exposing fields by default while still allowing developers to expose all fields if they prefer.